templates/showcase.html

```diff
@@ -5,12 +5,18 @@
 {% block copydoc %}https://docs.google.com/document/d/1daLrp_H_BGhSefvrP-aj3v3505dEz-zoKF2Mar2Dpt4/edit{% endblock %}
 
 {% block content %}
-<section id="main-content" class="p-strip--image is-dark vfio-hero">
+<section id="main-content" class="p-strip--suru-topped">
   <div class="row">
-    <h1 class="u-no-margin--bottom">Showcase</h1>
+    <div class="col-12">
```

Comment:
Technically that's what `u-fixed-width` is for (for situations when the row only has single 12 col child), but this it fine as well.